src/com/intellij/plugins/thrift/lang/psi/ThriftPrefixReference.java

```diff
@@ -28,4 +31,10 @@ public PsiElement resolve() {
   public Object[] getVariants() {
     return PsiElement.EMPTY_ARRAY;
   }
+
+  @NotNull
+  @Override
+  public Collection<? extends SymbolResolveResult> resolveReference() {
+    return super.resolveReference();
```

Comment:
Same comment as in `ThriftTypeReference.java`: is this required if it only calls `super.resolveReference()`?

Reply:
Same as above, get a compile error if not added.
`Error:(16, 8) java: com.intellij.plugins.thrift.lang.psi.ThriftPrefixReference is not abstract and does not override abstract method resolveReference() in com.intellij.model.SymbolReference`